eth/vm/computation.py

```diff
@@ -248,7 +277,7 @@ def memory_write(self, start_position: int, size: int, value: bytes) -> None:
 
     def memory_read(self, start_position: int, size: int) -> memoryview:
         """
-        Read and return a view of ``size`` bytes from memory starting at ``start_position``.
+        Read and return ``size`` bytes from memory starting at ``start_position``.
```

Comment:
```suggestion
        Read and return a view of ``size`` bytes from memory starting at ``start_position``.
```

This looks like a mistaken revert.